plugin/pkg/scheduler/schedulercache/node_info.go

```diff
@@ -108,6 +113,27 @@ func (n *NodeInfo) AllowedPodNumber() int {
 	return n.allowedPodNumber
 }
 
+func (n *NodeInfo) MemoryPressureCondition() api.ConditionStatus {
```

Comment:
Actually after thinking about it, I think it's much safer to return a value instead of pointer. We can potentially change it in the future.